dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/MetaDataApi.kt

```diff
@@ -0,0 +1,62 @@
+package org.dataland.datalandbackend.api
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.dataland.datalandbackend.model.DataMetaInformation
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+
+/**
+ * Defines the restful dataland-backend API regarding meta data searches.
+ */
+
+@RequestMapping("/metadata")
+interface MetaDataApi {
+
+    @Operation(
+        summary = "Search in Dataland for meta info about data.",
+        description = "Meta info about data sets registered by Dataland can be retrieved."
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Successfully retrieved meta info.")
+        ]
+    )
+    @GetMapping(
+        value = [""],
```

Comment:
`        value = [""],`
kann weg.

Reply:
und zwar auch in allen anderen Fällen (ist nicht nur hier der Fall gewesen)

Reply:
ok